src/components/Checkout/SocialShare.vue

```diff
@@ -0,0 +1,245 @@
+				<button class="social__btn social__btn--linkedin">
+					<icon-linkedin class="social__icon" />
+					<span>Share</span>
+				</button>
+				<button class="social__btn social__btn--email">
+					<span>Link to email</span>
+				</button>
+			</div>
+		</section>
+	</div>
+</template>
+
+<script>
+import IconFacebook from '@/assets/inline-svgs/social/facebook.svg';
+import IconLinkedin from '@/assets/inline-svgs/social/linkedin.svg';
+import IconTwitter from '@/assets/inline-svgs/social/twitter.svg';
+
+export default {
+	components: {
+		IconFacebook,
+		IconLinkedin,
+		IconTwitter
+	},
+	props: {
+		loans: {
+			type: Array,
+			required: true
+		}
+	},
+	data() {
+		return {
+			selectedLoanIndex: 0
+		};
+	},
+	computed: {
+		suggestedMessage: {
+			get() {
+				const selectedLoan = this.loans[this.selectedLoanIndex];
+				return `Kiva is an easy way to make a real difference in someone's life. Will you join me in helping ${selectedLoan.name} in ${selectedLoan.geocode.city} to pursue their dream?`; // eslint-disable-line max-len
+			},
+			set(val) {
+				return val;
+			}
+		},
+	},
+	methods: {
+		setSelectedLoanIndex(index) {
+			this.selectedLoanIndex = index;
+		},
+		printReceipt() {
+			if (typeof window !== 'undefined') {
+				window.print();
+			}
+		}
+	}
+};
+</script>
+
+<style lang="scss" scoped>
+@import 'settings';
+
+.thanks {
+	&__headline {
+		@include impact-text();
+
+		text-align: center;
+	}
+
+	&__subhead {
+		@include featured-text();
+
+		text-align: center;
+	}
+}
+
+.loans {
+	display: flex;
+	justify-content: space-between;
+
+	@include breakpoint(medium) {
+		justify-content: center;
+	}
+
+	@include breakpoint(large) {
+		flex-direction: column;
+		justify-content: space-between;
+		align-items: flex-end;
+	}
+
+	&__img {
+		opacity: 0.3;
+	}
+
+	&__circle {
+		width: rem-calc(70px);
+		height: rem-calc(70px);
+		border-radius: 50%;
+		overflow: hidden;
+
+		@include breakpoint(medium) {
+			margin: 0 0.5rem;
+		}
+
+		@include breakpoint(large) {
+			margin: 0;
+		}
+
+		&--selected {
+			.loans__img {
+				opacity: 1;
+			}
+		}
+	}
+}
+
+.social {
+	display: flex;
+	flex-wrap: wrap;
+
+	@include breakpoint(large) {
+		flex-direction: column;
+	}
+
+	&__btn {
+		display: flex;
+		align-items: center;
+		color: #fff;
+		width: calc(50% - 0.5rem);
+		margin: 0 1rem 1rem 0;
+		padding: 1rem;
+
+		&:nth-child(2n) {
+			margin-right: 0;
+		}
+
+		@include breakpoint(medium) {
+			width: calc(25% - 0.75rem);
+
+			&:nth-child(2n) {
+				margin-right: 1rem;
+			}
+
+			&:last-child {
+				margin-right: 0;
+			}
+		}
+
+		@include breakpoint(large) {
+			width: 100%;
+			margin-right: 0;
+
+			&:last-child {
+				margin-bottom: 0;
+			}
+		}
+
+		&--facebook {
+			background: #3b5998;
+		}
+
+		&--twitter {
+			background: #08a0e9;
+		}
+
+		&--linkedin {
+			background: #0077b5;
+		}
+
+		&--email {
+			color: $kiva-textlink;
+		}
+	}
+
+	&__icon {
+		width: rem-calc(24px);
+		height: rem-calc(24px);
```

Comment:
In the past when we've used rem-calc we leave off the px value. As long as this works I don't think it really matters, but thought I would mention it.

Reply:
Good to know, thanks!